test/image/mocks/scatter_fill_gradient_tonexty_toself.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Also this mock and new baseline should be named: `zz-scatter_fill_gradient_tonexty_toself`...